Libraries/TestHelpers.psm1

```diff
@@ -648,7 +648,7 @@ Function Run-LinuxCmd([string] $username,[string] $password,[string] $ip,[string
 					if ($timeOut)
 					{
 						$retValue = ""
-						Write-LogErr "Timeout while executing command : $MaskedCommand"
+						Throw "Calling function - $($MyInvocation.MyCommand). Timeout while executing command : $MaskedCommand"
```

Comment:
Is $MyInvocation.MyCommand different from $MaskedCommand?

Reply:
$MyInvocation.MyCommand is actually the function name